### ISSUE TITLE
About the loss formula

### ISSUE THREAD

**Issue by xushenkun** [analvikingur/pytorch_RVAE#4]
In rvae.py "loss = 79 * cross_entropy + kld_coef(i) * kld". 
Could you please explain why it is multiplied by 79 and what's the meaning of kld_coef(i)?

**Comment by kefirski** [comment]
[Here](https://github.com/kefirski/pytorch_RVAE/issues/3) you can find meaning of 79

**Comment by xushenkun** [comment]
@kefirski Thanks!
